gensim/test/test_utils.py

```diff
@@ -269,20 +269,26 @@ def hash_main(alg):
     import six
 
     assert six.PY3, 'this only works under Py3'
+    assert gensim.models.utils_any2vec.FAST_VERSION == 0, 'Cython extensions are broken'
```

Comment:
0 is fine, -1 is broken

Reply:
Yes, so if FAST_VERSION == 0, the assertion passes. If FAST_VERSION == -1, then the assertion fails with the specified message.

Reply:
Fast version can be eq 1 for example, better to >= 0

Reply:
Should we change this everywhere? There are other modules that rely on FAST_VERSION.

If we change it, may as well make it a boolean, right?

Reply:
> may as well make it a boolean
Definitely not. The particular version is important for debugging.